helm-charts/splunk-otel-collector/templates/config/_common.tpl

```diff
@@ -211,6 +211,34 @@ k8sattributes:
       {{- include "splunk-otel-collector.addExtraLabels" . | nindent 6 }}
     {{- end }}
 {{- end }}
+{{- define "splunk-otel-collector.k8sClusterReceiverAttributesProcessor" -}}
+k8sattributes/clusterReceiver:
+  pod_association:
+    - sources:
+      - from: resource_attribute
+        name: k8s.namespace.name
+    - sources:
+      - from: resource_attribute
+        name: k8s.node.name
+  extract:
+    metadata:
+      - k8s.namespace.name
+      - k8s.node.name
+      - k8s.pod.name
+      - k8s.pod.uid
+      - container.id
+      - container.image.name
+      - container.image.tag
+    annotations:
```

Comment:
Can you add another if statement to only include these annotations configurations if `splunk-otel-collector.splunkPlatformEnabled=true` since Splunk Observability cannot use these?

Reply:
done